android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/ListSectionProxy.java

```diff
@@ -675,12 +675,18 @@ public void populateViews(KrollDict data, TiBaseListViewItem cellContent, TiList
 				KrollDict properties = new KrollDict((HashMap) data.get(binding));
 				KrollDict diffProperties = viewItem.generateDiffProperties(properties);
 				if (!diffProperties.isEmpty()) {
+					if (view.getProxy() != null) {
+						view.getProxy().getProperties().putAll(diffProperties);
+					}
```

Comment:
The `getProperties()` method can return `null` if the proxy has been released. Can you add a `null` check please?

For your info, I'm the one who added the properties `null` check to the `composeContentDescription()` method in Titanium 8.0.1. The below ticket reproduces the crash. We should test this PR change with the "Don't keep activities" developer option too like the below ticket just in case. Thanks.
https://jira.appcelerator.org/browse/TIMOB-26989